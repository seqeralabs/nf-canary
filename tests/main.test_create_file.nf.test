nextflow_process {

    name "Test Process TEST_CREATE_FILE"
    script "main.nf"
    process "TEST_CREATE_FILE"

    test("TEST_CREATE_FILE") {

        when {
            process {
                """
                input[0] = 'dummy'
                """
            }
        }


        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out).match() }
            )
        }

    }

}
