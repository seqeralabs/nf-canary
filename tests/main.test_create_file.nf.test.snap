{
    "TEST_CREATE_FILE": {
        "content": [
            {
                "0": [
                    "test.txt:md5,d8e8fca2dc0f896fd7cb4cb0031ba249"
                ],
                "outfile": [
                    "test.txt:md5,d8e8fca2dc0f896fd7cb4cb0031ba249"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "23.10.1"
        },
        "timestamp": "2024-04-18T17:19:44.512229"
    }
}